docker/run-test

```diff
@@ -31,9 +59,22 @@ docker run --rm \
     export MTR_REPEAT='${MTR_REPEAT}'
     export MTR_VAULT_TOKEN='${MTR_VAULT_TOKEN}'
     export CI_FS_MTR='${CI_FS_MTR}'
+    export KEYRING_VAULT_MTR='${KEYRING_VAULT_MTR}'
+    export VAULT_V1_DEV_ADDRESS='http://vault-dev-v1:9200'
+    export VAULT_V2_DEV_ADDRESS='http://vault-dev-v2:9400'
+    export VAULT_V1_PROD_ADDRESS='https://vault-prod-v1:9300'
+    export VAULT_V2_PROD_ADDRESS='https://vault-prod-v2:9500'
+    export VAULT_V1_DEV_TOKEN='${VAULT_V1_DEV_TOKEN}'
```

Comment:
Here and everywhere.
I believe `VAULT_V1_DEV_TOKEN` / `VAULT_V2_DEV_TOKEN` should be renamed into `VAULT_V1_DEV_ROOT_TOKEN` / `VAULT_V2_DEV_ROOT_TOKEN` for consistency

Reply:
I agree, will be renamed